go.mod

```diff
@@ -0,0 +1,18 @@
+module go-google-scraper-challenge
+
+go 1.15
+
+require github.com/astaxie/beego v1.12.3
```

Comment:
Please refer to this Slack comment: https://nimble-co.slack.com/archives/C018FE1GPDG/p1607935898014600. https://github.com/beego/beego/  should be used instead.

Reply:
Actually, this seems to be the way to make it work form 🤯 Please refer to this Slack comment: https://nimble-co.slack.com/archives/C018FE1GPDG/p1608027390031900?thread_ts=1607935898.014600&cid=C018FE1GPDG

You can also check Andy's and Hoang's repository to check their working configuration.